dubbo-samples-boundary-test/dubbo-samples-hibernate/case-versions.conf

```diff
@@ -0,0 +1,24 @@
+#
+#
+#   Licensed to the Apache Software Foundation (ASF) under one or more
+#   contributor license agreements.  See the NOTICE file distributed with
+#   this work for additional information regarding copyright ownership.
+#   The ASF licenses this file to You under the Apache License, Version 2.0
+#   (the "License"); you may not use this file except in compliance with
+#   the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+
+# Supported component versions of the test case
+
+# Spring app
+dubbo.version=2.7.*, 3.*
+spring.version=4.*
```

Comment:
Not only test Spring 4.x, this example should have no special Spring version requirements and can use the wildcard *
```
spring.version=*
```

Reply:
> Not only test Spring 4.x, this example should have no special Spring version requirements and can use the wildcard *
> 
> ```
> spring.version=*
> ```



> > It doesn't use any new dubbo feature and it's also not a suggested best practice. that's why I put it in dubbo-samples-boundary-test folder
> 
> If this is the purpose, I prefer to change `dubbo-samples-boundary-test` to `integration-tests`

I'm ok with any name changing, @AlbumenJ what do you think

Reply:
It does have version requirement for spring. for both mybatis and hibernate.
they have different version requirement. if we need change spring version, we need modify hibernate version too.

Reply:
> > Not only test Spring 4.x, this example should have no special Spring version requirements and can use the wildcard *
> > ```
> > spring.version=*
> > ```
> 
> > > It doesn't use any new dubbo feature and it's also not a suggested best practice. that's why I put it in dubbo-samples-boundary-test folder
> > 
> > 
> > If this is the purpose, I prefer to change `dubbo-samples-boundary-test` to `integration-tests`
> 
> I'm ok with any name changing, @AlbumenJ what do you think

Looks fine. BTW, it is more important to add some description of this module.